src/main/java/webHandler/Capture.java

```diff
@@ -35,10 +55,23 @@ public Capture(String id, String rds, String s3, Date startTime, Date endTime) {
         this.status = "Running";
     }
 
+    public Capture(String id, String rds, String s3, Date startTime, Date endTime, int fileSizeLimit, int transactionLimit) {
```

Comment:
Any way we could overload these new constructors ? seems like a lot of duplicate code (DRY)